packages/@aws-cdk/aws-logs/test/integ.log-group.ts

```diff
@@ -0,0 +1,28 @@
+import { App, Stack, StackProps } from '@aws-cdk/core';
+import { IntegTest } from '@aws-cdk/integ-tests';
+import { LogGroup, DataProtectionPolicy } from '../lib';
+
+class LogGroupIntegStack extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    var audit = new LogGroup(this, 'LogGroupLambdaAudit', {
+      logGroupName: 'auditDestinationForCDK',
+    });
+
+    const dataProtectionPolicy = new DataProtectionPolicy({
+      identifiers: ['EmailAddress', 'DriversLicense-US'],
+      cloudWatchLogsAuditDestination: audit.logGroupName,
+    });
+
```

Comment:
This doesn't test enough of the functionality added here.

Reply:
Added additional fields in the test. Could not include delivery stream due to dependency issues, open to suggestions here.